site-pages/cart/cart.css

```diff
@@ -0,0 +1,113 @@
+.cart-container{
+    display: flex;
+    margin-top: 30px;
+}
```

Comment:
Just a suggestion to use rems

Reply:
Ohh thank you for this one. I totally forgot about it.